Horion/Module/Modules/Bhop.cpp

```diff
@@ -1,7 +1,16 @@
 #include "Bhop.h"
 
 Bhop::Bhop() : IModule(0, Category::MOVEMENT, "Hop around like a bunny!") {
+	registerEnumSetting("Bhop modes", &Bhopmode, 0);
+	Bhopmode = SettingEnum(this)
+				   .addEntry(EnumEntry("LowhopV2", 0))
```

Comment:
name it something else, no idea what the difference between lowhop and lowhopv2 is